geosys/bridge_api/default.py

```diff
@@ -118,7 +118,7 @@
 PGW_EXT = '.pgw'
 PNG_KMZ_EXT = '.png.kmz'
 PNG_EXT = '.png'
-TIFF_EXT = '.tif'
+TIFF_EXT = '.tiff'
```

Comment:
Doesn't this cause regression? Can we instead add a new extension constant for `tiff`.

Reply:
I don't think it will cause a regression as this is what's available in `v5`.
![image](https://github.com/user-attachments/assets/d2d557b8-9e3a-4d20-b694-8df80e6f71b6)